driver-sync/src/test/unit/com/mongodb/client/gridfs/GridFSUploadStreamSpecification.groovy

```diff
@@ -101,37 +101,54 @@ class GridFSUploadStreamSpecification extends Specification {
         def content = 'file content ' as byte[]
         def metadata = new Document('contentType', 'text/txt')
         def uploadStream = new GridFSUploadStreamImpl(clientSession, filesCollection, chunksCollection, fileId, filename, 255, metadata)
-        def chunksData
-        def fileData
+        def filesId = fileId
 
         when:
         uploadStream.write(content)
         uploadStream.close()
 
         then:
         if (clientSession != null) {
-            1 * chunksCollection.insertOne(clientSession) { Document data -> chunksData = data }
+            1 * chunksCollection.insertOne(clientSession) {
+                verifyAll(it, Document) {
```

Comment:
How does `verifyAll` work, as compared to what was here before?

Reply:
You can no longer assign to an outside variable, turns out there is an API method to use - verifyAll takes the passed in variable and a type for it.